validator/testcases/javascript/predefinedentities.py

```diff
@@ -121,8 +216,18 @@
          "value": {}},
     u"nsIBrowserSearchService":
         {"value":
-            {u"currentEngine": {"readonly": True},
-             u"defaultEngine": {"readonly": True}}},
+            {u"currentEngine":
+                {"readonly": search_warning(severity="high")},
```

Comment:
Shouldn't `readonly` be `True` or `False`? Should the keys here be `"dangerous"` instead?